plenum/server/config_req_handler.py

```diff
@@ -36,10 +36,22 @@ def doStaticValidation(self, request: Request):
         identifier, req_id, operation = request.identifier, request.reqId, request.operation
         typ = operation.get(TXN_TYPE)
 
-        if typ == TXN_AUTHOR_AGREEMENT_AML:
+        if typ == GET_TXN_AUTHOR_AGREEMENT:
+            parameters = [GET_TXN_AUTHOR_AGREEMENT_VERSION,
+                          GET_TXN_AUTHOR_AGREEMENT_DIGEST,
+                          GET_TXN_AUTHOR_AGREEMENT_TIMESTAMP]
+            parameters = [p for p in parameters if p in operation]
+            if len(parameters) > 1:
+                raise InvalidClientRequest(identifier, req_id,
+                                           "GET_TXN_AUTHOR_AGREEMENT request "
```

Comment:
I would add more detailed message about what combination is supported.
Something like `either 1 or no input parameters can be passed`.